src/worker/background.ts

```diff
@@ -0,0 +1,8 @@
+chrome.webRequest.onCompleted.addListener(
+    (details) => {
+      chrome.storage.sync.remove("os-access-token");
+    },
+    { urls: ["https://ibcwmlhcimymasokhgvn.supabase.co/auth/v1/logout"] }
```

Comment:
Maybe add a note here so we remember this has to be replaced later by a config key value.

Reply:
https://github.com/open-sauced/browser-extensions/issues/30